mule/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: mule
+    version: 1.2
+
+source:
+    fn: mule.tar.gz
+    url: https://github.com/SciTools/mule/releases/download/vn1.2/mule.tar.gz
+
+build:
+    number: 0
+    skip: True # [py3k or win]
```

Comment:
But I want it to ignore `py3k` on all platforms; I think it being an `and` would cause it to still try and test `py3k` on non-Windows platforms, wouldn't it?